libcxx/test/libcxx/utilities/intseq/intseq.binding/nodiscard.verify.cpp

```diff
@@ -0,0 +1,23 @@
+//===----------------------------------------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+// REQUIRES: std-at-least-c++26
+
+// <utility>
+
+// template<size_t I, class T, T... Values>
+//   [[nodiscard]] constexpr T get(integer_sequence<T, Values...>) noexcept;
+
+// check that get is marked [[nodiscard]]
+
+#include <utility>
+
+void test() {
+  std::index_sequence<1> seq;
+  get<0>(seq); // expected-warning {{ignoring return value of function}}
```

Comment:
I have recently added nodiscard tests in libcxx/utilities/intseq/nodiscard.verify.cpp Please move that line in there and remove this file.

```suggestion
#if TEST_STD_VER >= 26
  get<0>(seq); // expected-warning {{ignoring return value of function}}
#endif
```

Reply:
I've re-requested review since this is still marked as "changes requested" :)